### PR TITLE
Create my ToDo tasks widget

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [39745](https://github.com/demisto/etc/issues/39745)

## Description
- Created a new widget called 'My Todo Tasks' that presents a table with all todo tasks of the current user. 
  The widget was created as a script widget, which creates a table including all the open todo tasks of the current user and 
  returns it as a markdown table. 
  *The use of markdown format enables the Incident IDs in the table to be clickable, but prevents the possibility of setting 
    sort option to the SLA column.
  *Currently we don't have an option to make the todo task IDs clickable (opened an issue about it - [44307](https://github.com/demisto/etc/issues/44307)).

## Screenshots
64FC5F73-6A93-42E4-94B8-75889D2A7A19

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CommonWidgets/ReleaseNotes/1_2_2.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### New: MyToDoTasksWidget
+- Added **MyToDoTasksWidget** script that creates a table of all the ToDo tasks assigned to the current user. (Available from Cortex XSOAR 6.1.0).
+
+#### Widgets
+##### New: My ToDo Tasks
+- Added a **My ToDo Tasks** widget that presents a table of all the ToDo tasks assigned to the current user. (Available from Cortex XSOAR 6.1.0).
```
Comment (Itay4) [inline Packs/CommonWidgets/ReleaseNotes/1_2_2.md:8]:
```suggestion
- A widget that presents a table of all the ToDo tasks assigned to the current user. (Available from Cortex XSOAR 6.1.0).
```

File: Packs/CommonWidgets/ReleaseNotes/1_2_2.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### New: MyToDoTasksWidget
+- Added **MyToDoTasksWidget** script that creates a table of all the ToDo tasks assigned to the current user. (Available from Cortex XSOAR 6.1.0).
```
Comment (Itay4) [inline Packs/CommonWidgets/ReleaseNotes/1_2_2.md:4]:
```suggestion
- A script that creates a table of all the ToDo tasks assigned to the current user. (Available from Cortex XSOAR 6.1.0).
```

File: Packs/CommonWidgets/Scripts/MyToDoTasksWidget/MyToDoTasksWidget.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    if todo_tasks_query_res.get('statusCode') == 200:
+        todo_tasks = json.loads(todo_tasks_query_res.get('body', '{}'))
+        todo_tasks_data = todo_tasks.get('data', [])
+        for task in todo_tasks_data:
+            if task.get('status', '') == 'open':  # table includes only open tasks
+                title = task.get('title', '')
+                description = task.get('description', '')
+                task_id = task.get('id', '')
+                incident_id = task.get('incidentId', '')
+                clickable_incident_id = f'[{incident_id}]({os.path.join("#/Custom/caseinfoid", incident_id)})'
```
Comment (Itay4) [inline Packs/CommonWidgets/Scripts/MyToDoTasksWidget/MyToDoTasksWidget.py:30]: is `/Custom/caseinfoid` required? does it work properly in multi tenant? high availability?

Comment (ShacharKidor) [reply Packs/CommonWidgets/Scripts/MyToDoTasksWidget/MyToDoTasksWidget.py:30]: checked it on mt server and it worked fine.

**Review by Itay4 — APPROVED** [review]

**Comment by Itay4** [comment]
not adding test playbook due to sufficient coverage by the unit test
merging

**Pull request merged**